carvertical-hw/src/domains/controllers/authorizer.controller.ts

```diff
@@ -0,0 +1,30 @@
+import { APIGatewayRequestAuthorizerEvent, APIGatewaySimpleAuthorizerResult } from 'aws-lambda';
+
+import { userService } from '../services';
+
+export const authorizerController = async (event: APIGatewayRequestAuthorizerEvent): Promise<APIGatewaySimpleAuthorizerResult> => {
+    const basicAuth = event.headers?.authorization;
+	const userId = event.headers?.userid;
+
+	if (!basicAuth || !userId) {
+		return {
+			isAuthorized: false,
+		};
+	};
+
+	const base64Credentials = basicAuth.split(' ')[1];
```

Comment:
```suggestion
	const [, base64Credentials] = basicAuth.split(' ');
```
looks cleaner, right?